tests/lowMachNumberTests.cpp

```diff
@@ -588,6 +588,23 @@ INSTANTIATE_TEST_SUITE_P(
             .T_tExact = quadratic_T_t,
             .f0_v = f0_quadratic_v,
             .f0_w = f0_quadratic_w},
+        (LowMachMMSParameters){
+            .mpiTestParameter = {.nproc = 4,
+                                 .expectedOutputFile = "outputs/2d_tri_p2_p1_p1_nproc4",
+                                 .arguments = "-dm_plex_separate_marker -dm_refine 2 "
+                                              "-vel_petscspace_degree 2 -pres_petscspace_degree 1 -temp_petscspace_degree 1 "
+                                              "-dmts_check .001 -ts_max_steps 4 -ts_dt 0.1 "
+                                              "-ksp_type fgmres -ksp_gmres_restart 10 -ksp_rtol 1.0e-9 -ksp_error_if_not_converged "
+                                              "-pc_type fieldsplit -pc_fieldsplit_0_fields 0,2 -pc_fieldsplit_1_fields 1 -pc_fieldsplit_type schur -pc_fieldsplit_schur_factorization_type full "
+                                              "-fieldsplit_0_pc_type lu "
+                                              "-fieldsplit_pressure_ksp_rtol 1e-10 -fieldsplit_pressure_pc_type jacobi -dm_distribute"},
```

Comment:
I would put this up on line 594 with the other DM options

Reply:
@knepley Thanks!  I've cleaned up the dm_distribute and reduced the refine level to speed up the tests